### ISSUE TITLE
Loosing the wheels

### ISSUE THREAD

**Issue by pawel-ziecina** [carla-simulator/carla#1104]
Hi, below is method for "unscreewing the wheels" in most (all?) vehicles.

```python
import carla
import random
c = carla.Client('localhost', 2000)
w = c.get_world()
vbs = w.get_blueprint_library().filter('vehicle*')
vb = random.choice(vbs)
sp = w.get_map().get_spawn_points()[0]
v = w.spawn_actor(vb, sp)
sp.rotation.pitch = -90
sp.location.z = 40
w.get_spectator().set_transform(sp)
v.set_simulate_physics(False)
v.apply_control(carla.VehicleControl(throttle=10.0))
v.set_simulate_physics(True)
```

OS: Ubuntu 18.04
Carla: 0.9.2 from official release

**Comment by pawel-ziecina** [comment]
It was just play with API. I don't claim it is logic/common sequence of operations.

You are right when disabled physics before appling control to car it wouldn't move.

But when you enable physics again - car body will move, while wheels will stay in place. I think it is a bug.

![screenshhot_190107-164606](https://user-images.githubusercontent.com/1385347/50777409-cbcdbe00-129b-11e9-8d4b-5e7b2df850f6.png)

**Comment by tanglrHello** [comment]
This seems to be the same problem I reported in #1001 . It has been fixed in release 0.9.2.

**Comment by pawel-ziecina** [comment]
You are right - it is the same issue.

Confirming that I'm using official 0.9.2 build.

![screenshhot_190108-085747](https://user-images.githubusercontent.com/1385347/50817248-83f27980-1323-11e9-8841-d334e73a6fb3.png)

**Comment by nsubiron** [comment]
I think it is related but different than #1001, only some vehicles were loosing the wheels and it was due a corruption of their physics asset in UE4. That was already fixed in 0.9.2. Not sure what's causing this, we should investigate.

**Comment by pawel-ziecina** [comment]
Could not reproduce on 0.9.4.
